# Message to control high level actions of the Ego-Vehicle


# lane_change : Change lane (0 keep lane; 1 lane change right; 2 lane change left)

int16 lane_change
int16 accel
